Source/Core/Core/HW/MemoryInterface.cpp

```diff
@@ -169,15 +163,15 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
   mmio->Register(base | MI_UNKNOWN1, MMIO::DirectRead<u16>(&g_mi_mem.unknown1),
                  MMIO::DirectWrite<u16>(&g_mi_mem.unknown1));
 
-  // The naming is confusing here: the registed contains the lower part of
+  // The naming is confusing here: the register contains the lower part of
   // the address (hence MI_..._LO but this is still the high part of the
   // overall register.
   mmio->Register(base | MI_PROT_ADDR_LO, MMIO::DirectRead<u16>(&g_mi_mem.prot_addr.hi),
                  MMIO::DirectWrite<u16>(&g_mi_mem.prot_addr.hi));
   mmio->Register(base | MI_PROT_ADDR_HI, MMIO::DirectRead<u16>(&g_mi_mem.prot_addr.lo),
                  MMIO::DirectWrite<u16>(&g_mi_mem.prot_addr.lo));
 
-  for (int i = 0; i < 10; ++i)
+  for (u32 i = 0; i < g_mi_mem.timers.size(); ++i)
```

Comment:
```
for (std::size_t i = 0; i < g_mi_mem.timers.size(); ++i)
```
?

Reply:
This would make the formed addresses 64-bit, which would cause truncations

Reply:
Ok, I guess then cast?

```
for (u32 i = 0; i < static_cast<u32>(g_mi_mem.timers.size()); ++i)
```

Reply:
I mean, it's fine as is, since they're the same signedness and there's no loss of information.